section-engineering-education/engineering-education
# Testimonial carousel

**Description:**
Add a testimonial slider to the EngEd homepage. Utilized Hugos Data template to hold the content for each testimonial.

**How to Test:**
Navigate to beta.section.io/engineering-education/ Or pull down this branch and view locally. The design should match the screenshot below:
<img width="1791" alt="image" src="https://user-images.githubusercontent.com/15935329/165160953-14f2369e-b139-4e8f-8489-1aa2d3876b25.png">
